op-chain-ops/genesis/config.go

```diff
@@ -204,6 +210,17 @@ type DeployConfig struct {
 	// RequiredProtocolVersion indicates the protocol version that
 	// nodes are recommended to adopt, to stay in sync with the network.
 	RecommendedProtocolVersion params.ProtocolVersion `json:"recommendedProtocolVersion"`
+	// SuperchainConfigInitiator represents the address of the Security Council role.
+	SuperchainConfigInitiator common.Address `json:"superchainConfigInitiator"`
+	// SuperchainConfigVetoer represents the address of the SuperchainConfigVetoer role.
+	SuperchainConfigVetoer common.Address `json:"superchainConfigVetoer"`
+	// SuperchainConfigDelay represents the time after which an upgrade can occur ifit
+	SuperchainConfigDelay uint64 `json:"superchainConfigDelay"`
+	// SuperchainConfigMaxPause represents the time after which an upgrade can occur ifit
+	SuperchainConfigMaxPause uint64 `json:"superchainConfigMaxPause"`
+	// SequencerKeys represents the set of allowed Sequencer key pairs (batcher hash and unsafe
+	// block signer).
+	SuperchainConfigSequencerKeyPairs []SequencerKeyPair `json:"superchainConfigSequencerKeyPairs"`
 }
```

Comment:
We should be sure to either log warnings in `Check` or error. If they must be set, then we should error

Reply:
adding that